### PR TITLE
Cumulus ansible log refactor

### PR DESCRIPTION
**Pull request by kotfic**
Implements a test for ansible logging and fixes logging to use ```get_post_logger``` API

### CODE REVIEW

**Comment by kotfic** [comment]
This should address  https://github.com/Kitware/cumulus/pull/197

File: tests/cases/ansible_run_log_test.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        self.webserver.start()
+        time.sleep(0.3)
+
+    def tearDown(self):
+        os.remove(self.requests_file)
+        self.webserver.terminate()
+        self.webserver.join()
+
+        pass
+
```
Comment (cjh1) [inline tests/cases/ansible_run_log_test.py]: Left over pass?

File: tests/cases/ansible_run_log_test.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+import json
+
+def flaskProcess(requests_file):
+    from flask import g
+    from flask import Flask
+    from flask import request
+
+    import logging
+    from logging.handlers import SysLogHandler
+
```
Comment (cjh1) [inline tests/cases/ansible_run_log_test.py]: Logging to the syslog?

**Comment by cjh1** [comment]
LGTM

**Pull request merged**